app/assets/stylesheets/application.scss

```diff
@@ -39,7 +39,7 @@
   }
 
   .panel {
-    background:rgba(255,255,255,0.6);
+    background:rgba(255,255,255,0.30);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space